client/flutter/pubspec.lock

```diff
@@ -305,7 +319,7 @@ packages:
       name: test_api
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.2.15"
+    version: "0.2.11"
```

Comment:
I don't think this was supposed to decrease?

Reply:
No, it wasn't intended.